codegen-server/python/src/main/kotlin/software/amazon/smithy/rust/codegen/server/python/smithy/generators/PythonServerModuleGenerator.kt

```diff
@@ -163,6 +164,22 @@ class PythonServerModuleGenerator(
         )
     }
 
+    private fun RustWriter.renderPyLambdaTypes() {
+        rustTemplate(
+            """
+            let lambda_ = #{pyo3}::types::PyModule::new(py, "lambda_")?;
+            lambda_.add_class::<#{SmithyPython}::lambda::PyLambdaContext>()?;
+            pyo3::py_run!(
+                py,
+                lambda_,
+                "import sys; sys.modules['$libName.lambda_'] = lambda_"
+            );
+            m.add_submodule(lambda_)?;
+            """,
+            *codegenScope,
+        )
+    }
```

Comment:
I am not sure I understand why the module is suffixed with underscore..

Reply:
Right! I propose we call this module `aws_lambda`. WDYT?

Reply:
Yes makes sense, addressed in https://github.com/awslabs/smithy-rs/pull/1985/commits/75bd902168f572e99b6d998ca2f46bc4f94ddda4